<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterationMainWindow</class>
 <widget class="QMainWindow" name="RegisterationMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RegisterationMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 41, 61);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>522</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="newManagerAction"/>
    <addaction name="openManagerAction"/>
   </widget>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>&amp;Controls</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="registerAction"/>
    <addaction name="deleteManagerAction"/>
    <addaction name="actionSortAndFilterTable"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAboutRegistrationManager"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControls"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="newManagerAction"/>
   <addaction name="openManagerAction"/>
   <addaction name="separator"/>
   <addaction name="registerAction"/>
   <addaction name="actionSortAndFiterMode"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newManagerAction">
   <property name="text">
    <string>New Manager</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Start New Manager Registration Manager &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Ctrl+N&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="registerAction">
   <property name="text">
    <string>Register</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Click to start your registration&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Alt+N&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Alt+N</string>
   </property>
  </action>
  <action name="openManagerAction">
   <property name="text">
    <string>&amp;Open Manager</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Open a registration manager&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Ctrl+O&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="deleteManagerAction">
   <property name="text">
    <string>Delete Manager</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Delete Manager&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="actionAboutRegistrationManager">
   <property name="text">
    <string>About Registration Manager</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionSortAndFilterTable">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Filter.png</normaloff>:/images/images/Filter.png</iconset>
   </property>
   <property name="text">
    <string>Sort And Filter Table</string>
   </property>
   <property name="toolTip">
    <string>Sort and filter data in the table</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="actionSortAndFiterMode">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Filter.png</normaloff>:/images/images/Filter.png</iconset>
   </property>
   <property name="text">
    <string>Sort and Fiter Mode</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#000000;&quot;&gt;Table is in sort and filter mode.&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#000000;&quot;&gt;Click to remove sort and filter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
